resources/js/index.js

```diff
@@ -1,22 +1,22 @@
 var list_task = [
   {
-    id: Date.now(),
+    id: Date.now() + 45,
     message: "Hola",
     due_date: new Date("2019-05-18"),
     creation_date: new Date(),
     priority: false,
     complete: false
   },
   {
-    id: Date.now(),
+    id: Date.now() + 34,
```

Comment:
Why +34? 👀

Reply:
Because when JS is executed and our id is a number generate by .now(), our 3 elements had the same id, because of that we add any number to avoid issue when work with CRUD